library/templates/dcp_stateassemblydistricts.yml

```diff
@@ -0,0 +1,36 @@
+dataset:
+  name: &name dcp_stateassemblydistricts
+  version: "{{ version }}"
+  acl: public-read
+
+  source:
+    url:
+      path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyad_{{ version }}.zip
+      subpath: nyad_{{ version }}/nyad.shp
+    geometry:
+      SRS: EPSG:4326
+      type: MULTIPOLYGON
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+  
+  destination:
+    name: *name
+    geometry:
+      SRS: EPSG:4326
+      type: MULTIPOLYGON
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      ### ESRI Shapefile
+      New York State Assembly District boundaries for the City of New York (Clipped to shoreline). 
+      These district boundaries represent the redistricting as of the US Census 2010.
+      Source will be updated once the 2020 Census data is available.
```

Comment:
Maybe "Source will be updated once the redistricted in response to the 2020 Census"

Reply:
Or maybe "...once the district boundaries get redistricted..."?